docs/dapp_overview.md

```diff
@@ -69,7 +69,7 @@ The overall communication in between Nodes can be seen as going up or down the t
 ![node-communication](img/node-comm.jpg)
 
 ## What are Aliases?
-As shown in the image above, Nodes communicate with each other throught it's parent dApp Channels. This implies that the communication is one-layered only, because the parent dApp, by definition, isn't a Node and therefore isn't able to send or receive messages (it only defines where it's child Nodes can read and write from):
+As shown in the image above, Nodes communicate with each other thought it's parent dApp Channels. This implies that the communication is one-layered only, because the parent dApp, by definition, isn't a Node and therefore isn't able to send or receive messages (it only defines where it's child Nodes can read and write from):
```

Comment:
wouldn't it be `through its`? thought is an idea or opinion produced by thinking, or occurring suddenly in the mind.

Reply:
that's correct

Reply:
@ptcar2009 not sure if mean @nicholaspcr was correct. or the text.
I believe he is right. Should be `through its`